<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="3" ignored="0" total="3" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="896" started-at="2017-12-20T18:05:44Z" finished-at="2017-12-20T18:05:45Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="896" started-at="2017-12-20T18:05:44Z" finished-at="2017-12-20T18:05:45Z">
      <class name="example.Testng">
        <test-method status="FAIL" signature="Get(org.apache.poi.ss.usermodel.Sheet, org.apache.poi.ss.usermodel.Sheet, int, org.apache.poi.ss.usermodel.Workbook)[pri:0, instance:example.Testng@3abbfa04]" name="Get" duration-ms="0" started-at="2017-12-20T23:35:44Z" finished-at="2017-12-20T23:35:44Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Cannot inject @Test annotated Method [Get] with [interface org.apache.poi.ss.usermodel.Sheet, interface org.apache.poi.ss.usermodel.Sheet, int, interface org.apache.poi.ss.usermodel.Workbook].
For more information on native dependency injection please refer to http://testng.org/doc/documentation-main.html#native-dependency-injection]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Cannot inject @Test annotated Method [Get] with [interface org.apache.poi.ss.usermodel.Sheet, interface org.apache.poi.ss.usermodel.Sheet, int, interface org.apache.poi.ss.usermodel.Workbook].
For more information on native dependency injection please refer to http://testng.org/doc/documentation-main.html#native-dependency-injection
	at org.testng.internal.Parameters.checkParameterTypes(Parameters.java:334)
	at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:274)
	at org.testng.internal.Parameters.createParameters(Parameters.java:562)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:672)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1316)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1044)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1134)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Get -->
        <test-method status="FAIL" signature="Run()[pri:0, instance:example.Testng@3abbfa04]" name="Run" duration-ms="859" started-at="2017-12-20T23:35:44Z" finished-at="2017-12-20T23:35:45Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at example.Testng.Get(Testng.java:53)
	at example.Testng.Run(Testng.java:41)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Run -->
        <test-method status="FAIL" signature="Set(java.util.ArrayList, org.apache.poi.ss.usermodel.Sheet, org.apache.poi.ss.usermodel.Workbook)[pri:0, instance:example.Testng@3abbfa04]" name="Set" duration-ms="0" started-at="2017-12-20T23:35:45Z" finished-at="2017-12-20T23:35:45Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Cannot inject @Test annotated Method [Set] with [class java.util.ArrayList, interface org.apache.poi.ss.usermodel.Sheet, interface org.apache.poi.ss.usermodel.Workbook].
For more information on native dependency injection please refer to http://testng.org/doc/documentation-main.html#native-dependency-injection]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Cannot inject @Test annotated Method [Set] with [class java.util.ArrayList, interface org.apache.poi.ss.usermodel.Sheet, interface org.apache.poi.ss.usermodel.Workbook].
For more information on native dependency injection please refer to http://testng.org/doc/documentation-main.html#native-dependency-injection
	at org.testng.internal.Parameters.checkParameterTypes(Parameters.java:334)
	at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:274)
	at org.testng.internal.Parameters.createParameters(Parameters.java:562)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:672)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1316)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1044)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1134)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Set -->
      </class> <!-- example.Testng -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
